Arrowgene.Ddon.Shared/Entity/PacketStructure/S2CPawnTrainingSetTrainingStatusRes.cs

```diff
@@ -0,0 +1,25 @@
+using Arrowgene.Buffers;
+using Arrowgene.Ddon.Shared.Network;
+
+namespace Arrowgene.Ddon.Shared.Entity.PacketStructure
+{
+    public class S2CPawnTrainingSetTrainingStatusRes : ServerResponse
+    {
+        public override PacketId Id => PacketId.S2C_PAWN_TRAINING_SET_TRAINING_STATUS_RES;
+
+        public class Serializer : PacketEntitySerializer<S2CPawnTrainingSetTrainingStatusRes>
+        {
+            public override void Write(IBuffer buffer, S2CPawnTrainingSetTrainingStatusRes obj)
+            {
+                WriteServerResponse(buffer, obj);
```

Comment:
This one really has no fields?

Reply:
Only the default response error and result fields declared in the superclass for confirmation that setting the training status did work